armi/cases/case.py

```diff
@@ -936,6 +936,8 @@ def copyInterfaceInputs(
                 except NonexistentSetting(key):
                     runLog.debug(f"{key} is not a valid setting; continuing on anyway.")
                     label = key
+            else:
```

Comment:
I'm not sure about this approach. I'll try to find time this week to test and discuss this.

Reply:
Okay, I added the check below, like we discussed.

Reply:
nice!